test/fitnesse/junit/FitNesseRunnerExtensionTest.java

```diff
@@ -27,4 +40,35 @@ protected FitNesseContext createContext(Class<?> suiteClass) throws Exception {
       return super.createContext(suiteClass);
     }
   }
+
+  public static class ListenerExtension extends JUnitRunNotifierResultsListener {
```

Comment:
I actually used it to test (with debugger) that my behavior was called. I apologize for such a rude form of testing. I could have just deleted it.